### PR TITLE
Pandoc not rendering tables correctly

### PR DESCRIPTION
**Pull request by sdwheeler**
See Issue #2228 

Changed the Pandoc options to specify GFM instead of Markdown. This fixed the table issue but now the Yaml header is being rendered improperly in the output TXT file. I added a function to the appveyor script to strip the Yaml header off the about_topic.md file. I tested this with Pandoc 2.0.6 and 2.2. 

Version(s) of document impacted
------------------------------
- [ ] Impacts 6.1 document
- [x] Impacts 6.0 document
- [ ] Impacts 5.1 document
- [ ] Impacts 5.0 document
- [ ] Impacts 4.0 document
- [ ] Impacts 3.0 document

Reason(s) for not updating all version of documents
--------------------------------------------------
- [ ] The documented feature was introduced in version (list version here) of PowerShell
- [ ] This issue only shows up in version (list version(s) here) of the document
- [ ] This PR partially fixes the issue, and issue #<insert here> tracks the remaining work

### CODE REVIEW

**Comment by zjalexander** [comment]
### :x: Validation status: errors


**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by sdwheeler** [comment]
Close and reopen to retrigger the build.

**Comment by zjalexander** [comment]
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[appveyor.ps1](https://github.com/sdwheeler/PowerShell-Docs/blob/sdw-pandoc/appveyor.ps1) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2018%5C5%5C7%5Cbab27324-4c59-a348-1473-9c6d587e4020%5CPullRequest%5C201805072203574603-2410%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=3SL3ho4D9gwuv4g%2F8U6%2B4jrzMzNNpKKviAgLREZ6tak%3D&st=2018-05-07T22%3A08%3A42Z&se=2018-06-07T22%3A13%3A42Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

File: appveyor.ps1 [hunk 1]
```diff
@@ -7,6 +7,33 @@ param(
...
 
+function Get-ContentWithoutHeader {
+    param(
+      $path
+    )
+
+    $doc = Get-Content $path -Encoding UTF8
+    $start = $end = -1
+
+    for ($x = 0; $x -lt 30; $x++) {
```
Comment (adityapatwardhan) [inline appveyor.ps1]: Can you add a comment why 30 was chosen?

Comment (sdwheeler) [reply appveyor.ps1]: No Yaml header is 30 lines long so that should be plenty. Added comment to the code.

**Comment by zjalexander** [comment]
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[appveyor.ps1](https://github.com/sdwheeler/PowerShell-Docs/blob/sdw-pandoc/appveyor.ps1) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2018%5C5%5C7%5Cbab27324-4c59-a348-1473-9c6d587e4020%5CPullRequest%5C201805072343226464-2410%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=0bP8x6lu5Opj8QfSTjebBkLHjK3bIB%2FwXN30j0BRyD4%3D&st=2018-05-07T23%3A47%3A53Z&se=2018-06-07T23%3A52%3A53Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Pull request merged**